### DISCUSSION TITLE
Include htmx.org and hyperscript.org packages as part of javascript bundle

### DISCUSSION THREAD

**Discussion by i-am-mike-davis** [fief-dev/fief#431] [Ideas]
# About your idea

The Administration UI requires htmx.org and hyperscript.org
from unpkg.com. Without these packages, all htmx functions
on the UI break (e.g. adding redirect URIs). 

# What motivates it 

I like Fief and I want it to function even if it cannot access unpkg.com in case of:
- Strict Firewall Administration
- Airgapped Networks
- Unpkg.com going down
- Local network interruption

# How you plan to implement it before you start working

Ask npm to kindly include the packages.
Include packages as part of Rollup bundle

# Pull Request

https://github.com/fief-dev/fief/pull/430

**Comment by fief-bailiff** [comment]
Hail, @i-am-mike-davis 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to [star our repository](https://github.com/fief-dev/fief) ⭐️

<p align="center">
<img width="656" alt="star-fief" src="https://github.com/fief-dev/fief/assets/1144727/b2db7f59-102f-47e5-9751-67fa10372c52">
</p>

## Want to support us?

Subscribe to one of our paid plan to help us continue our work and receive exclusive information and benefits! Starts at $5/month 🪙

<p align="center">
    <a href="https://polar.sh/fief-dev">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/embed/subscribe.svg?org=fief-dev&darkmode=1">
      <img alt="Subscribe" src="https://polar.sh/embed/subscribe.svg?org=fief-dev">
    </picture>
    </a>
</p>

Farewell!

**Comment by i-am-mike-davis** [comment]
Updated Pull Request https://github.com/fief-dev/fief/pull/430
